lib/web/apiserver.go

```diff
@@ -3125,3 +3128,27 @@ func SSOSetWebSessionAndRedirectURL(w http.ResponseWriter, r *http.Request, resp
 
 	return nil
 }
+
+// authExportPublic returns the CA Certs that can be used to set up a chain of trust which includes the current Teleport Cluster
+//
+// GET /webapi/sites/:site/auth/export?type=<auth type>
```

Comment:
@marcoandredinis this just came up in conversation and I happened to notice that we put the API behind `webapi/sites/:site`, but we never consult the `site` parameter.

This has two side effects:

1. `curl example.teleport.com/webapi/sites/somefakethingthatdoesntexist/auth/export` works
2. This can never be used to get the CA for trusted clusters

If we're only ever looking at _this_ cluster, we probably shouldn't have put it behind `/sites`. Alternatively, we can augment this to actually look at the site that's specified.

Reply:
What do you think about adding a new path `webapi/auth/export` and only exporting the current cluster's CA Certs?

Otherwise, and because this is a public endpoint, we could be leaking trusted cluster information (I don't think it's a sensitive value, however, the less information we return on public endpoints the better)